schemes/rrtmgp/rrtmgp_lw_aerosols.F90

```diff
@@ -0,0 +1,37 @@
+!> \file rrtmgp_lw_aerosols.F90
+!!
+
+!> This module contains the call to the RRTMGP-lw radiation routine
```

Comment:
I don't think this comment is correct.  Instead maybe just replace it with a comment stating that here is where the RRTMGP aerosol LW optical depth is set?